Lundalogik/lime-elements
# build(npm): switch to using registry.npmjs.org

How to test:
Only regression testing needed. Everything should work like before.

If `package-lock.json` is changed when running `npm install`, discard the changes, remove the `node_modules` folder, and try again.
